### ISSUE TITLE
After deleting row isn't after deleting row

### ISSUE THREAD

**Issue by kotchuprik** [unclead/yii2-multiple-input#71]
Hi! Looks like event fires before input has been removed.

```js
function reindex() {
  $('.list-cell__order input').each(function (index, item) { 
    $(item).val(index);
  });
}

$('#addressform-models').on('afterAddRow', function (e) {
  reindex();
}).on('afterDeleteRow', function () {
  reindex();
});
```

[![https://gyazo.com/436b45d3f1d91477be938c8e1808ec3d](https://i.gyazo.com/436b45d3f1d91477be938c8e1808ec3d.gif)](https://gyazo.com/436b45d3f1d91477be938c8e1808ec3d)

**Comment by unclead** [comment]
@kotchuprik  Thx for reporting, i will fix it soon.

**Comment by kotchuprik** [comment]
thank you man! :)

**Comment by unclead** [comment]
Should be fixed via 13c343b1818e28164eebf50e9630af87d114b86b

**Comment by kotchuprik** [comment]
thanks, will check today
